### ISSUE TITLE
invalid Do not use prefix `_` for a variable that is used.

### ISSUE THREAD

**Issue by justin808** [sds/slim-lint#3]
slim-lint app/views

```
app/views/devise/confirmations/new.html.slim:3 [W] RuboCop: Lint/UnderscorePrefixedVariableName: Do not use prefix `_` for a variable that is used.
```
I don't see any `_` variable

```
h2
  | Resend confirmation instructions
= simple_form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f|
  = f.error_notification
  = f.full_error :confirmation_token
  .form-inputs
    = f.input :email, required: true, autofocus: true
  .form-actions
    = f.button :submit, "Resend confirmation instructions"
= render "devise/shared/links"
```

**Comment by sds** [comment]
Hey @justin808, thanks for the report.

Yes, this is a bug in how we're extracting the Ruby code from the Slim template. It turns out an assumption was made that isn't valid. I'm working on a solution but a proper fix may take me some time, so for the moment you can disregard these nonsensical errors. Sorry about the confusion!

**Comment by sds** [comment]
I've released version [0.4.0](https://github.com/sds/slim-lint/releases/tag/v0.4.0) which should address a number of these issues. Let me know if you see other strange/erroneous errors!

**Comment by justin808** [comment]
@sds Looks good

Any opinions on my config file?

```
linters:
  LineLength:
    max: 120
  ConsecutiveControlStatements:
    max_consecutive: 6

  RuboCop:
    ignored_cops:
      - Lint/BlockAlignment
      - Lint/EndAlignment
      - Lint/Void
      - Metrics/LineLength
      - Style/AlignParameters
      - Style/BlockNesting
      - Style/FileName
      - Style/IfUnlessModifier
      - Style/IndentationConsistency
      - Style/IndentationWidth
      - Style/Next
      - Style/TrailingBlankLines
      - Style/TrailingWhitespace
      - Style/WhileUntilModifier
      - Metrics/BlockNesting

```

**Comment by sds** [comment]
Hey @justin808,

Every development team has its opinions, so I can't speak to your settings for `LineLength`.

For `ConsecutiveControlStatements`, I believe the reason you had to set this so high was due to a bug that has since been fixed in 1ccea479e30814041ae39629c6e011fe7a9de815. You can probably stick with the default of 3.

For `RuboCop`, bear in mind that when you explicitly set `ignored_cops` you overwrite the defaults in `config/default.yml`. This list has changed with version 0.4.0, and it looks like the only cop you are explicitly adding to the list is `Metrics/BlockNesting`. Do you think this cop should be ignored by default? My general preference would be to use `.rubocop.yml` to disable cops. If there is a cop that is annoying or erroneous in the context of a Slim template, we should add it to the default ignore list.

**Comment by justin808** [comment]
@sds Thanks for the feedback. What's your preference for line length? I typically use 27" cinema screens and would have my team use the same.

**Comment by sds** [comment]
My personal preference is 100 since (for me) it:

* Captures the "spirit" of the 80 column limit, while accepting that there are many times where code can be made slightly more readable if you allow this rule to be broken in certain scenarios
* Anything over 100 is usually a sign of complexity, and should be broken down further (e.g. with partials)

Again, highly subjective.
